src/engraving/libmscore/note.h

```diff
@@ -252,6 +274,8 @@ class Note final : public EngravingItem
 
     bool sameVoiceKerningLimited() const override { return true; }
 
+    std::vector<LineAttachPoint> _lineAttachPoints;
```

Comment:
let's maybe add new elements already with "m_" prefix? or we keep them old for old files?

Reply:
All the other names have just the underscore prefix, so I would leave like this for consistency at the moment. Maybe at some point we make a separate PR and refactor all of them with m_ prefix